torch/_dynamo/variables/nn_module.py

```diff
@@ -537,23 +527,55 @@ def call_method(
         args: "List[VariableTracker]",
         kwargs: "Dict[str, VariableTracker]",
     ) -> "VariableTracker":
+        # circular import.. resolve it later
         from .builder import VariableBuilder
 
-        options = VariableTracker.propagate(self, args, kwargs.values())
+        def wrap_values(items, getsource=AttrSource):
```

Comment:
@jansel @voznesenskym @mlazos I want to get some directional guidance.

I think i'm trying to change the default behavior to not specialize on NNModule and to trace through them.

As an implementation detail i've been modifying UnspecNN... and directing traffic to it, thinking that later i'd rip out NNModule and rename UnspecNN to  NNModule.

As I was hacking through bugs, I started copying stuff from NNModule that i thought was harmless, such as this wrap_values helper for helping implement .children() below.

I initially hoped to make children (and many other nn.Module methods) fall through `inline_user_function_return` path, but I hit issues where a set was being constructed and assuming constant variables, which the children module members are not.  So I started copying the stuff from NNModule...

1) am I right that I should not do this, as it essentially specializes again on these child modules? and instead i _must_ use inlining translator and find other ways around these bugs?

2) any suggestion on the set/const thing?